api/test/gateways/Academy-Benefits/FetchCustomerAPI.test.js

```diff
@@ -0,0 +1,108 @@
+const AcademyBenefitsFetchRecordAPI = require('../../../lib/gateways/Academy-Benefits/FetchCustomerAPI');
+const nock = require('nock');
+
+describe('AcademyBenefitsFetchRecord gateway', () => {
+  let db;
+  let logger;
+
+  const dbError = new Error('Database error');
```

Comment:
Did you need to create the error in this test when you don't test the response to errors in this set?

Reply:
I'm a bit confused with the question, do you mind explaining a bit more what you mean?

Reply:
You're creating a `new Error()` constant on this line, but never using it anywhere in the code. I think you should either test for the error being thrown, or remove this constant

Reply:
Ah, yeh that's correct its being used in fetchRecord.test and not the other two, have removed them

Reply:
@louiseswift I've made the changes recommended please have a look and let me know what you think